### DISCUSSION TITLE
How to update the openobserve version

### DISCUSSION THREAD

**Discussion by jiujiu994** [openobserve/openobserve#4920] [Q&A]
How to update the openobserve version, steps for updating the openobserve version，

**`Local mode` in `sqlite`**

**Comment by prabhatsharma** [comment]
Download the new version and replace the binary.

**Reply by jiujiu994** [reply]
Sometimes data structure errors may be reported,

After replacement, need to run the command?

./openobserve migrate-meta -f sqlite -t sqlite
./openobserve migrate-file-list -f sqlite -t sqlite

**Reply by hengfeiyang** [reply]
Which version do you upgrade to which version?

**Reply by jiujiu994** [reply]
> Which version do you upgrade to which version?

v0.10.8-rc2 to v0.12.1

**Reply by hengfeiyang** [reply]
For these version, no need migration, did you get some error?

**Reply by jiujiu994** [reply]
> For these version, no need migration, did you get some error?

I encountered it during the previous version upgrade.  I'm worried that I might also encounter it this time,Under what circumstances is migration necessary?

**Reply by hengfeiyang** [reply]
For single node (SQLite) actually it always work without migration. if you got any error please tell us or create new issue.

**Reply by jiujiu994** [reply]
> For single node (SQLite) actually it always work without migration. if you got any error please tell us or create new issue.
Well, thank you

**Reply by jiujiu994** [reply]
> For single node (SQLite) actually it always work without migration. if you got any error please tell us or create new issue.

So, does the version upgrade directly replace the binary?

**Reply by hengfeiyang** [reply]
Yeap, download new binary, stop old binary, replace it, start again.

**Reply by jiujiu994** [reply]
> Yeap, download new binary, stop old binary, replace it, start again.

tks
